examples/junit/src/test/java/com/example/LifecycleFuzzTest.java

```diff
@@ -42,6 +49,7 @@ static void beforeAll() {
   @BeforeEach
   void beforeEach() {
     beforeEachCount++;
+    beforeEachCalled = true;
```

Comment:
Why is `beforeEachCalled` needed? We already have `beforeEachCount` to check that `beforeEach` is called.

Reply:
I renamed the variables to clarify their purpose. The difference is that one is static whereas the other is an instance variable.